src/game/dot_prod_world/dot_zero.lean

```diff
@@ -5,6 +5,9 @@ namespace tuple -- hide
 /- 
 # Vector world
 
+Strategy:
+Think about how we can use zero_dot to prove dot_zero.
```

Comment:
Maybe hint at using dot_comm? Something like "Think about how we can use zero_dot and other lemmas to prove dot_zero"?